gulpfile.js

```diff
@@ -115,16 +115,39 @@ gulp.task('build/zone-node.js', ['compile-esm-node'], function(cb) {
 
 // Zone for the browser.
 gulp.task('build/zone.js', ['compile-esm'], function(cb) {
```

Comment:
what's the difference between build/zone.js and build/zone-legacy.js? I don't quite follow.. can you please improve the comments and maybe even document the build artifacts/docs in the repo so that it's clear what the individual files in the npm distributions do? thanks!

Reply:
@IgorMinar, thanks for the review.
- `zone.js` is the full package, include `evergreen` and `legacy` support.
- `zone-legacy` only contains the `legacy support`.
- `zone-evergreen` contains the `evergreen support`.

So `zone.js` = `zone-legacy` + `zone-evergreen`.

Reply:
can you document this in readme.md and in the gulpfile.js? thanks